config.json.example

```diff
@@ -1,7 +1,11 @@
 {
   "submission_key_fpr": "65A1B5FF195B56353CC63DFFCC40EF1228271441",
   "hidserv": {
-    "hostname": "avgfxawdn6c3coe3.onion",
+    "hostname": "avgfxawdn6c3coe3",
     "key": "Il8Xas7uf6rjtc0LxYwhrx"
+  },
+  "hidservv3": {
+    "hostname": "ADD_YOUR_VALUE",
+    "key": "ADD_YOUR_VALUE"
```

Comment:
Does the Tor hidden service v3 support multiple key formats for v3 ATHS? The key format is hardcoded in https://github.com/freedomofpress/securedrop-workstation/pull/381/files#diff-95ca75b6cf53e335d99d46567e37e9d5R1 , I am wondering if we should expose it to the config

Reply:
This is the actual key value, and we are generating it in the vm as required.